Src/DocGen/Distributed-Docs-Files/HostDocs.bat

```
@@ -0,0 +1,158 @@
+@ECHO OFF
```

Not a fan of shell scripts, they're notoriously hard to maintain and tend to silently fail.

---

Yeah.. In retrospect, I wish I did it in PowerShell. First I thought it would be much shorter and then I ended up with this. :)
Either way, it needs to be Windows, so either this or PowerShell. ))